webserver/views/api/v1/dataset_eval.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import absolute_import
+from flask import Blueprint, jsonify, request
+from flask_login import current_user
+from webserver.decorators import auth_required
+import db.dataset_eval
+
+bp_dataset_eval = Blueprint('api_v1_dataset_eval', __name__)
+
+@bp_dataset_eval.route("/pending-jobs-for-user", methods=["GET"])
+@auth_required
+def pending_jobs_for_user():
+    """Return a list of pending jobs for a user to be evaluated remotely"""
+    jobs = db.dataset_eval.get_pending_jobs_for_user(current_user.id)
+    jobs_list = []
+    for job in jobs:
+        jobs_list.append(job["id"])
+    return jsonify(
+        success=True,
+        jobs=jobs_list
```

Comment:
@kartikgupta0909 If we asked the user to add their name to the config file then there is nothing stopping them from making a mistake. Since we can already get this data from the API, I think it makes more sense to return it there.
Returning additional metadata can help us during development/debugging of the remote application. For example, we can log this information when we retrieve it.

Following on from @gentlecat's ideas, it would make sense to make all related endpoints return the same data. So, we could have a single endpoint with parameters, which we also use for the website, or separate endpoints which return specific data. In both cases, it makes sense to be consistent in what we return.